e2e-tests/src/test/high_latency.rs

```diff
@@ -0,0 +1,86 @@
+use std::cmp::max;
+
+use aleph_client::{
+    pallets::session::SessionApi,
+    waiting::{AlephWaiting, BlockStatus},
+};
+use log::info;
+
+use crate::{config::setup_test, synthetic_network::set_out_latency};
+
+#[tokio::test]
+pub async fn high_out_latency() -> anyhow::Result<()> {
```

Comment:
I think a doc description of what this test does and why this is relevant would be beneficial here.

Reply:
added